### PR TITLE
Filter

### PR DESCRIPTION
**Pull request by ethn**
(not ready to be merged!)

### CODE REVIEW

File: mod/filter/set/abstract/browse_filter_form.rb [hunk 1]
```diff
@@ -24,7 +24,23 @@ def filter_wql
...
+    hash[:id] = ["not in", not_ids]
+  end
+  hash
+end
+
+def blocked_ids
+  not_ids = filter_param :not_ids
+  return unless not_ids.present?
+  not_ids.split ","
 end
```
Comment (ethn) [inline mod/filter/set/abstract/browse_filter_form.rb:44]: above needs to be moved to decko eventually

Comment (ethn) [reply mod/filter/set/abstract/browse_filter_form.rb:44]: (except the type_id part)

File: mod/metrics/spec/set/right/downvotee_search_spec.rb [hunk 2]
```diff
@@ -8,33 +8,33 @@
...
+                              right_plus: apple.name,
+                              limit: 3
+        vote_down_metrics metrics
+        downvotee_search_card = apple.fetch trait: [:metric, :downvotee_search]
+        result = downvotee_search_card.format.get_search_result
+        expect(Card[result[0]].id).to eq(metrics[0].id)
+        expect(Card[result[1]].id).to eq(metrics[2].id)
+        expect(Card[result[2]].id).to eq(metrics[1].id)
+      end
+    end
```
Comment (ethn) [inline mod/metrics/spec/set/right/downvotee_search_spec.rb:21]: I was just trying to comment out a bit of noise (`puts metrics`) but then to pass code climate I had to refactor :)

File: mod/wikirate/set/type/wikirate_company.rb [hunk 3]
```diff
@@ -21,9 +21,9 @@ def headquarters_jurisdiction_code
 end
 
 view :listing do
-  # TODO: 1. move this structure to code under "browse_item" view
-  #       2. create a more appropriate listing structure
-  _render_content structure: "browse company item"
+  wrap_with :div, class: "border p-2" do
+    _render_thumbnail_no_link
+  end
```
Comment (ethn) [inline mod/wikirate/set/type/wikirate_company.rb:26]: would be a more natural analogue to the metric `listing` view if this had a company count.

File: mod/metrics/set/type_plus_right/metric/formula/wiki_rating.rb [hunk 4]
```diff
@@ -2,7 +2,7 @@
   view :rating_core do |args|
     table_content =
       card.translation_table.map do |metric, weight|
-        [subformat(metric)._render_thumbnail_plain(args), weight + "%"]
+        [subformat(metric)._render_thumbnail_plain(args), "#{weight}%"]
```
Comment (xithan) [inline mod/metrics/set/type_plus_right/metric/formula/wiki_rating.rb:5]: better use `nest` instead of `subformat`

**Review by xithan — APPROVED** [review]

File: mod/metrics/set/type_plus_right/metric/formula/wiki_rating.rb (hunk 4 above)
Comment (ethn) [reply mod/metrics/set/type_plus_right/metric/formula/wiki_rating.rb:5]: ok, I'll change.  I wasn't really wanting to tinker in this area.  not sure why this broke in tests.

**Pull request merged**